ignite/pkg/gocmd/gocmd.go

```diff
@@ -139,6 +146,15 @@ func Install(ctx context.Context, path string, pkgs []string, options ...exec.Op
 	return exec.Exec(ctx, command, append(options, exec.StepOption(step.Workdir(path)))...)
 }
 
+// IsInstallError returns true if err is interpreted as a go install error.
+func IsInstallError(err error) bool {
+	if err == nil {
+		return false
+	}
+	const label = "no required module provides package"
+	return strings.Contains(err.Error(), label)
```

Comment:
Maybe the extra variable could be removed in this case:
```suggestion
	return strings.Contains(err.Error(), "no required module provides package")
```